clippy_lints/src/transmute/transmute_null_to_fn.rs

```diff
@@ -0,0 +1,61 @@
+use clippy_utils::consts::{constant, Constant};
+use clippy_utils::diagnostics::span_lint_and_then;
+use clippy_utils::{is_integer_literal, is_path_diagnostic_item};
+use rustc_hir::{Expr, ExprKind};
+use rustc_lint::LateContext;
+use rustc_middle::ty::Ty;
+use rustc_span::symbol::sym;
+
+use super::TRANSMUTE_NULL_TO_FN;
+
+const LINT_MSG: &str = "transmuting a known null pointer into a function pointer";
```

Comment:
I think we can inline those consts now that we use them only once.

Reply:
Done